config/settings.py

```diff
@@ -151,18 +155,21 @@
 # Database
 # https://docs.djangoproject.com/en/2.0/ref/settings/#databases
 
+POSTGRES_PASSWORD = env('POSTGRES_PASSWORD')
```

Comment:
This is up to the team... but just including defaults would allow for anyone to pull the project to simply 
`python manage.py <<cmd>>`

Given they have a local postgres instance (or a docker container running with the appropriate port exposed). Would just remove one additional step in the onboarding process.